analyzers/ruby/integration_test.go

```diff
@@ -0,0 +1,111 @@
+package ruby_test
+
+import (
+	"flag"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/apex/log"
+	"github.com/fossas/fossa-cli/analyzers"
+	"github.com/fossas/fossa-cli/exec"
+	"github.com/fossas/fossa-cli/files"
+	"github.com/fossas/fossa-cli/module"
+	"github.com/fossas/fossa-cli/pkg"
+	"github.com/fossas/fossa-cli/testing/fixtures"
+	"github.com/fossas/fossa-cli/testing/runfossa"
+	"github.com/stretchr/testify/assert"
+)
+
+var rubyAnalyzerFixtureDir = filepath.Join(fixtures.Directory(), "ruby", "analyzer")
+
+func TestMain(m *testing.M) {
```

Comment:
Be aware that `TestMain` is _package_-scoped, not _file_-scoped. This will prevent unit tests from running on `-short`, just like what's happening for Node.JS right now.